src/lib/get-configuration/get-configuration.test.js

```diff
@@ -1,14 +1,16 @@
-const { getConfiguration } = require('./config');
-
 jest.spyOn(console, 'warn').mockImplementation(() => {});
-jest.mock('chalk', () => {
+jest.unstable_mockModule('chalk', () => {
```

Comment:
Jest isn't quite there with ESM. Mocking a module being one outstanding pain point. Based on a few recommendations, we use this module-specific mock (available since Jest v27), then lazy import the file we want to test _after_ the mock is in place.

https://www.sitepen.com/blog/jest-esm-almost-there